modules/post/windows/gather/enum_av_excluded.rb

```diff
@@ -0,0 +1,148 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'rex'
+
+class Metasploit3 < Msf::Post
+  include Msf::Post::Windows::Registry
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name'          => 'Windows Antivirus Excluded Locations Enumeration',
+        'Description'   => 'This module will enumerate all excluded directories within supported AV products',
+        'License'       => MSF_LICENSE,
+        'Author'        => [
+          'Andrew Smith', # original metasploit module
+          'Jon Hart <jon_hart[at]rapid7.com>' # improved metasploit module
+        ],
+        'Platform'      => [ 'win' ],
+        'SessionTypes'  => [ 'meterpreter' ]
```

Comment:
Since this module seems to rip stuff out of the registry, could this not work on `cmd` sessions too?